doc/base/classes.xml

```diff
@@ -79,7 +79,7 @@
 				Returns the arc sine of 's' in radians. Use to get the angle of sine 's'.
 				[codeblock]
 				# s is 0.523599 or 30 degrees if converted with rad2deg(s)
-				s = asin(0.5)				
+				s = asin(0.5)
```

Comment:
shouldn't this be
`var s = asin(0.5)`

Reply:
no var is intentionally left out to keep things simple. s could be declared elsewhere.